[2025-07-15T21:16:53.208+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-15T21:16:53.239+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projeto_Joao_Motta.etapa_1 manual__2025-07-15T21:16:49.720013+00:00 [queued]>
[2025-07-15T21:16:53.249+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projeto_Joao_Motta.etapa_1 manual__2025-07-15T21:16:49.720013+00:00 [queued]>
[2025-07-15T21:16:53.252+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-15T21:16:53.270+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): etapa_1> on 2025-07-15 21:16:49.720013+00:00
[2025-07-15T21:16:53.282+0000] {standard_task_runner.py:63} INFO - Started process 55 to run task
[2025-07-15T21:16:53.293+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Projeto_Joao_Motta', 'etapa_1', 'manual__2025-07-15T21:16:49.720013+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/DAG 2.py', '--cfg-path', '/tmp/tmplptv1hjd']
[2025-07-15T21:16:53.299+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask etapa_1
[2025-07-15T21:16:53.329+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-07-15T21:16:53.483+0000] {task_command.py:426} INFO - Running <TaskInstance: Projeto_Joao_Motta.etapa_1 manual__2025-07-15T21:16:49.720013+00:00 [running]> on host e1948b1953b5
[2025-07-15T21:16:54.939+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projeto_Joao_Motta' AIRFLOW_CTX_TASK_ID='etapa_1' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T21:16:49.720013+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-15T21:16:49.720013+00:00'
[2025-07-15T21:16:54.940+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-15T21:16:54.954+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-15T21:16:54.955+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/DAG 2.py", line 17, in etapa_1
    os.mkdir(caminho)
FileNotFoundError: [Errno 2] No such file or directory: 'C:/Dados - João Gabriel Motta'
[2025-07-15T21:16:54.970+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=Projeto_Joao_Motta, task_id=etapa_1, execution_date=20250715T211649, start_date=20250715T211653, end_date=20250715T211654
[2025-07-15T21:16:54.985+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 10 for task etapa_1 ([Errno 2] No such file or directory: 'C:/Dados - João Gabriel Motta'; 55)
[2025-07-15T21:16:55.016+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-15T21:16:55.046+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-15T21:16:55.052+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
